### ISSUE TITLE
Perfect Dark PC port changes my HDR DTS brightness

### ISSUE THREAD

**Issue by Dreamboum** [fgsfdsfgs/perfect_dark#568]
It's unbearable! I recently acquired an HDR compatible screen, and when I launch Perfect Dark PC Port, it changes the HDR brightness. So, I always have to go to the Windows 11 menu to put the HDR DTS brightness back as it should.

It doesn't happen to me in other games, but only in Perfect Dark PC Port. If I run it, it puts on sloppy brightness.

**Comment by fgsfdsfgs** [comment]
The port is not doing anything specifically to change brightness, so I don't know what is causing that. Unfortunately I don't have an HDR monitor either, so I can't check.

**Comment by LonelySpaceDetective** [comment]
Does playing it in a window (or fullscreen, if you were playing it in a window beforehand) make any difference?
Also, if you were to change HDR brightness while the game is still running, does it stick or does switching back to the game mess with the brightness again?

**Comment by fgsfdsfgs** [comment]
From what I can gather, Windows is supposed to have a separate brightness setting for SDR apps in HDR mode, so maybe that just defaults to a really really low value?
According to Graslu you also can just turn HDR on and back off to reset it to normal instead of manually adjusting brightness.

**Comment by Dreamboum** [comment]
I think the PC port of Perfect Dark is not HDR compatible. When I launch the game, it deactivates it.

**Comment by fgsfdsfgs** [comment]
Yes, it isn't. But I don't know why it would reset brightness. Maybe it's just how Windows HDR works and we can't really do anything about it.
